src/components/frame/v5/pages/FundsPage/context/IncomingFundsLoadingContextProvider.tsx

```diff
@@ -6,23 +6,44 @@ import React, {
   type PropsWithChildren,
 } from 'react';
 
+import useFundsStateUpdater, { useRefetchColonyData } from './hooks.ts';
```

Comment:
Slightly confusing imports here, I would just export them both as default inside an object to avoid this default / non-default importing from a generic hooks file

Reply:
Regularly I do named exports, but my blindness totally made me miss this 🫠 I will fix it asap!